stdlib/tests/crypto/stark/mod.rs

```diff
@@ -9,6 +9,7 @@ use verifier_recursive::{generate_advice_inputs, VerifierData};
 
 // Note: Changes to MidenVM may cause this test to fail when some of the assumptions documented
 // in `stdlib/asm/crypto/stark/verifier.masm` are violated.
+#[ignore = "depends on horner_eval_*"]
```

Comment:
Are we planning on merging with this `#[ignore]` still there, and removing it in #1656? Or rebasing #1656 on top of this one, removing the `#[ignore]` there, and then merging both at once?

Reply:
Once, this and #1656 are merged, there should be another PR updating the MASM code to remove the above `#[ignore]`.